.github/workflows/sphinx-build.yml

```diff
@@ -47,12 +59,21 @@ jobs:
       with:
         python-version: 3.8
 
+    - name: Sphinx Gallery Cache
+      uses: actions/cache@v2
+      env:
+        cache-name: sphx-glry-doctest
+      with:
+        path: |
+          tutorial/MNIST
+          docs/source/tutorial
+        key: py3.8-${{ env.cache-name }}-${{ hashFiles('tutorial/**/README.rst', 'tutorial/**/*.py') }}
```

Comment:
Just curious but is there a reason behind not matching all files under `tutorial`?

Reply:
Because I don't think `tutorial` is going to have files other than `.rst` and `.py`.

Reply:
I might be overthinking (we might be include asset-like files or modify directories, etc. in the future) but if that's the reason, I think it's safer to just include everything. It's not all too obvious to go check this file when making changes to the tutorial.